repos/system_upgrade/el7toel8/actors/migratentp/libraries/migratentp.py

```diff
@@ -36,12 +32,13 @@ def ntp2chrony(root, ntp_conf, step_tickers):
         ntp_configuration = ntp2chrony.NtpConfiguration(root, ntp_conf, step_tickers)
         ntp_configuration.write_chrony_configuration('/etc/chrony.conf', '/etc/chrony.keys',
                                                      False, True)
-    except Exception as e:
-        raise StopActorExecutionError('ntp2chrony failed: {}'.format(e))
+    except (FileExistsError, FileNotFoundError, PermissionError) as e:
```

Comment:
> pylint now fails with python-2 due to unknown exceptions. Any suggestions?

Usually we go with `OSError` (all mentioned exceptions are made from `OSError`) which is suggested way when writting Py2 & Py3 compatible code.

```suggestion
    except OSError as e:
```

Another possibility is to add comments like below (and you hit problem with too long line which you need then handle as well..), but I suggest to go with the `OSError` as mentioned above. Showing this possibility just for completness:
```suggestion
    # NOTE: this code is executed by Python3 only where mentioned exceptions are defined. Ignore pylint for Py2 here
    except (FileExistsError, FileNotFoundError, PermissionError) as e:  # noqa: E0602; pylint: disable-undefined-variable
```

Reply:
Thank you. Hopefully pylint is happy now.

Reply:
You are welcome \o seems good